src/Umbraco.Web.UI.NetCore/Umbraco.Web.UI.NetCore.csproj

```diff
@@ -16,6 +16,7 @@
     <ProjectReference Include="../Umbraco.Web.Common/Umbraco.Web.Common.csproj" />
     <ProjectReference Include="../Umbraco.Web.Website/Umbraco.Web.Website.csproj" />
     <ProjectReference Include="../Umbraco.Persistence.SqlCe/Umbraco.Persistence.SqlCe.csproj" Condition="'$(OS)' == 'Windows_NT'" />
+    <ProjectReference Include="../Umbraco.TestData/Umbraco.TestData.csproj" Condition="'$(Configuration)' == 'Debug'" />
```

Comment:
This seems strange.. Why do we need this project reference?

Reply:
Good question.  There was a note in the readme for the test data project, saying that: _Currently this project is referenced in the Umbraco.Web.UI project but only when it's being built in Debug mode (i.e. when testing within Visual Studio)._.

So I was attempting to replicate that setup for V9.

But now I check, there doesn't seem to be this project reference in V8, so I guess the readme note is out of date.

Hence have removed.